src/Bridges/bridge_optimizer.jl

```diff
@@ -1496,6 +1496,23 @@ function MOI.get(
     return unbridged_function(b, func)
 end
 
+function MOI.get(
+    b::AbstractBridgeOptimizer,
+    attr::MOI.ConstraintPrimal,
+    ci::MOI.ConstraintIndex,
```

Comment:
We could restrict this to scalar functions and sets, there should be no issue with vector ones

Reply:
Ah yes, cool cool. I'll make the change